react/postcss.config.cjs

```diff
@@ -2,13 +2,13 @@
 
 module.exports = {
   plugins: [
+    require('postcss-nesting'),
     require('postcss-preset-env')({
       stage: false,
       features: {
         'custom-media-queries': true,
         'media-query-ranges': true,
         'custom-selectors': true,
-        'nesting-rules': true,
```

Comment:
**Concern.**

I think this app is not correctly installing dependencies for PostCSS.

**TL;DR:** Install `postcss-preset-env`, not `postcss-nested`, because you use the former and it uses (thus isntalls, the latter). And install at [v9](https://www.npmjs.com/package/postcss-preset-env?activeTab=versions), not [v7](https://github.com/TACC-Cloud/hazmapper/blob/task/WG-246-wrapper-modal-layout/react/package-lock.json#L3263), in hopes that the warnings are resolved.

<details>

1. `postcss-preset-env` already installs `postcss-nesting`, and one simply need "turn it on" as was being done.
2. If `npm install` / `npm ci` warnings originate from nested dependencies, report to package maintainer(s), so every user benefits.

_P.S. Article ["How to fix NPM dependency hell" @ "Path to resolution"](https://sylhare.github.io/2022/02/09/How-to-fix-npm-dependency-library-hell.html#path-to-resolution) looks pretty good._

</details>

Reply:
npm install postcss-preset-env@latest --save-dev

npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: hazmapper@0.0.0
npm WARN Found: postcss-preset-env@7.8.3
npm WARN node_modules/postcss-preset-env
npm WARN   peer postcss-preset-env@"^7.8.3" from @tacc/core-styles@2.24.1
npm WARN   node_modules/@tacc/core-styles
npm WARN     @tacc/core-styles@"^2.24.1" from the root project
npm WARN   1 more (the root project)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer postcss-preset-env@"^7.8.3" from @tacc/core-styles@2.24.1
npm WARN node_modules/@tacc/core-styles
npm WARN   @tacc/core-styles@"^2.24.1" from the root project

npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: hazmapper@0.0.0
npm ERR! Found: postcss-preset-env@9.4.0
npm ERR! node_modules/postcss-preset-env
npm ERR!   dev postcss-preset-env@"^9.4.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer postcss-preset-env@"^7.8.3" from @tacc/core-styles@2.24.1
npm ERR! node_modules/@tacc/core-styles
npm ERR!   @tacc/core-styles@"^2.24.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

I still get this resolve warning and error when trying to npm install with v9.

Reply:
Well, fart. Okay. Thanks for trying. I can't argue well enough to spend more time on this. It's an architectural "maybe we should do this" thing, not a blocker.